src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -5055,6 +5055,9 @@
 /debug:pdbonly                    Emit PDB file only.
 /deterministic                    Produce a deterministic assembly
                                   (including module version GUID and timestamp)
+/instrument:options
+                               Sets instrumentation options.
```

Comment:
Start description on the previous line (as it is done for other switches) and match the alignment.